src/debug-adapter/nativeScriptPathTransformer.ts

```diff
@@ -42,6 +42,21 @@ export class NativeScriptPathTransformer extends UrlPathTransformer {
 
         relativePath = relativePath.replace('tns_modules', nodePath);
 
+        const pathToNsconfig = path.join(webRoot, 'nsconfig.json');
```

Comment:
this method is called for every file on the device, so reading and parsing may slow the debugging. This config won't be changed during the debugging. We may set it a single time, like the platform.